### ISSUE TITLE
state.isValid?

### ISSUE THREAD

**Issue by etobia** [grommet/react-formify#3]
Thanks for releasing this! Quick question on the state and errors arguments. Is it possible to check whether the `state.isValid` or if there are any errors (e.g. `errors.any`). I'm struggling to implement an error summary at the top of the form. I tried checking the `errors` object for empty, but it always seems to have a `get` function, even when there are no errors.

**Comment by atanasster** [comment]
There is a isValid, but in FormState, thus inaccessible from your code.
You can check for any errors in the same way:
!Object.keys(errors).length

**Comment by alansouzati** [comment]
This is now available in 0.7.0, sorry for the delay on this.

```javascript
const UserForm = () => (
  <Form
    defaultErrors={{ email: 'This is an existing email' }}
    defaultValue={{ email: 'alan@gmail.com' }}
    onSubmit={user => alert(JSON.stringify(user, null, 2))}
    rules={rules}
  >
    {(state, errors, isValid) => (
      <fieldset>
        <input {...state.name} />
        {errors.name ? <span className="error">{errors.name}</span> : undefined}
        <input {...state.email} />
        <input {...state.get('optional.field')} />
        {errors.email ? <span className="error">{errors.email}</span> : undefined}
        <button type='submit'>Add</button>
        <button
          type='button'
          onClick={() => {
            state.set('name', 'Alan Test');
            state.set({
              email: 'alan@test.com',
            });
          }}
          disabled={isValid}
        >
          Set Default
        </button>
      </fieldset>
    )}
  </Form>
);
```

Notice that the `isValid` has been added as an additional argument to the children function
